src/Cache/Cache.php

```diff
@@ -98,7 +98,7 @@ public static function fromJson($json)
             ));
         }
 
-        $requiredKeys = array(
+        static $requiredKeys = array(
```

Comment:
For my understanding: are you referencing to packed array optimizations in PHP7+?